G/ginkgo/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ginkgo"
+version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/youwuyou/ginkgo.git", "c4128a21b1114bedfc19153a9508b2bd3b54954f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd ginkgo/
+mkdir build && cd build
+cmake -DGINKGO_BUILD_TESTS=OFF -DGINKGO_BUILD_BENCHMARKS=ON -DGINKGO_BUILD_EXAMPLES=ON -DGINKGO_DOC_GENERATE_EXAMPLES=OFF -G "Ninja" ../
+ninja -j${nproc} 
+DESTDIR=${WORKSPACE}/destdir/shared ninja install
```

Comment:
If anything this should be
```suggestion
DESTDIR=${prefix}/shared ninja install
```
but why changing the prefix at all?